cmd/argot/cli/cli.go

```diff
@@ -118,7 +118,7 @@ func Run(flags tools.CommonFlags) {
 
 func seekConfig(configPath string, args []string) (*config.Config, bool, error) {
 	var err error
-	pConfig := &config.Config{} // empty default config
+	pConfig := config.NewDefault()
```

Comment:
Honestly if there's no config file, I'd rather have it be an error. Is there any case where a user would want the default config?

Reply:
I typically use it to quickly inspect aliasing in a function, or see what the dataflow summary is. Usually it doesn't require setting any non-default option. I think it makes sense for the cli but less for the other tools.